tools/x-docs/plugins/gatsby-plugin-doc-pages/gatsby-node.js

```diff
@@ -97,6 +122,24 @@ exports.createPages = ({ boundActionCreators, graphql }) => {
 			return Promise.reject(result.errors);
 		}
 
+		const dupes = getDuplicateNodeKeys(result.data.allMarkdownRemark.edges, 'frontmatter.path')
+		if(dupes.size) {
+			for(const [url, nodes] of dupes.entries()) {
+				const filePaths = nodes.map(
+					({fileAbsolutePath}) => '  ◆ ' + chalk.cyan(path.relative(repoBase, fileAbsolutePath))
+				).join('\n');
+
+				console.warn();
+				console.warn(`${chalk.black.keyword('black').bold.bgYellow(' WARNING ')} there are multiple files that would have the url ${chalk.blue.underline(url)}:
+
+${filePaths}
+
+which file is chosen is arbitrary. you should move of the files to another folder.
```

Comment:
Maybe reword this to something like:

"This will cause unexpected output, please re-name or move the files to resolve the conflict"